README.md

```diff
@@ -1 +1,30 @@
-# graal-patches
\ No newline at end of file
+# Graal Patches
+This project builds and publishes the artifact graal-sdk.tar.gz required by Conclave. This module is a separate and
```

Comment:
Please update this README in light of it now being part of a separate repo. Try to read it from the PoV of a developer who doesn't know what this repo does and give them the information they need to build our fork.